pulpcore/cli/common/context.py

```diff
@@ -701,7 +716,7 @@ class PulpRepositoryContext(PulpEntityContext):
 
     ENTITY = _("repository")
     ENTITIES = _("repositories")
-    HREF_PATTERN = r"repositories/(?P<plugin>\w+)/(?P<resource_type>\w+)/"
+    HREF_PATTERN = r"repositories/(?P<plugin>\w+)/(?P<resource_type>[\w\-_]+)/"
```

Comment:
This was required for pulp_container where we have the `/repositories/container/container-push/` endpoint.

Reply:
Can plugins have dashes too? I'd update the in all cases.

Reply:
I added both the dash and underscore to all default regexes.